server/routes/service-provider/appointment/post-session-feedback/attendance/postSessionAttendanceFeedbackPresenterComponent.ts

```diff
@@ -0,0 +1,78 @@
+import PresenterUtils from '../../../../../utils/presenterUtils'
+import { SummaryListItem } from '../../../../../utils/summaryList'
+import DateUtils from '../../../../../utils/dateUtils'
+import { FormValidationError } from '../../../../../utils/formValidationError'
+import { AppointmentDeliveryType } from '../../../../../models/appointmentDeliveryType'
+import Address from '../../../../../models/address'
+import SessionFeedback from '../../../../../models/sessionFeedback'
+
+interface PostSessionAttendanceFeedbackFormText {
+  title: string
+  subTitle: string
+  attendanceQuestion: string
+  attendanceQuestionHint: string
+  additionalAttendanceInformationLabel: string
+}
+interface PostSessionAttendanceAppointmentDetails {
```

Comment:
I think ideally we should make an ActionPlanAppointment extend Appointment interface.
The only issue is that the Appointment interface doesn't have nullable fields whilst ActionPlanAppointments do.

I think therefor we should change the Appointment interface to have nullable fields, however I was thinking of doing this after the PR. Unless anyone feels strongly for me to do it here.

Reply:
yeh i think the issue is that on the backend, the action plan appointments are not fully integrated with the newer generic appointment interfaces. we definitely need to do some work to tidy it all up and ensure there is minimal repeated code between SA appointments and feedback and AP appointments and feedback.

Reply:
I had a brief discussion with Ryan on slack - we think it might be useful to mention it in the dev chat on Monday